src/common/regions/region_db.ts

```diff
@@ -1,4 +1,7 @@
-import { sortBy, takeRight, values, Dictionary } from 'lodash';
+import sortBy from 'lodash/sortBy';
+import takeRight from 'lodash/takeRight';
+import values from 'lodash/values';
+import type { Dictionary } from 'lodash';
```

Comment:
ooooh, I didn't know there was `import type`...  I wonder if that helps with the next.js thing you noticed?